Add information about uiRegistry component searching by properties (#4336)
magento/devdocs: guides/v2.2/ui_comp_guide/concepts/ui_comp_uiregistry.md

var component = registry.get('%componentName%');
```

> The `uiRegistry` instance allows to search the component by some property. 

Reviewer: ```suggestion
The `uiRegistry` instance allows you to search for components by property values. 
```

> If you know some unique property of the component that you need to find, you can use the following code to get this component:

Reviewer: ```suggestion
If you know a unique property of a component that you need to find, you can use the following code to get the component:
```

```js
var component = registry.get('%property% = %propertyValue%');
```

> **Example of usages**

Reviewer: ```suggestion
**Examples**
```

> The code below lists how to get a component by a full component name:

Reviewer: ```suggestion
The following code shows how to get a component by a full component name:
```

```js
// Admin > Products > Catalog > Add Product
var fieldName = registry.get('product_form.product_form.product-details.container_name.name');
```

> The code below lists how to get a component by some property value:

Reviewer: ```suggestion
The following code shows how to get a component by a property value:
```

```js
// Admin > Products > Catalog > Add Product
var fieldName = registry.get('index = name');
// or 
fieldName = registry.get('inputName = product[name]');
```

> By this code, you will obtain the same JS component as with using the full component name.

Reviewer: ```suggestion
The previous example gets the same JS component as using the full component name.
```
